thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/route.js

```diff
@@ -453,9 +453,8 @@ export default Route.extend({
    * @param {Array} anomalyIds - the IDs of anomalies that have been reported for this alert.
    * @return {undefined}
    */
-  loadAnomalyData: task(function * (anomalyIds) {
-    yield timeout(300);
-    const hasDimensions = this.currentModel.exploreDimensions && this.currentModel.exploreDimensions.length;
+  loadAnomalyData: task(function * (anomalyIds, exploreDimensions) {
```

Comment:
Does this still need to be a task since you remove the `yield` keyword?

Reply:
+1

Reply:
Yes - it does yield for child tasks and a data processing function to complete.